tests/admin_views/tests.py

```diff
@@ -1465,6 +1492,14 @@ def test_login(self):
         self.assertFalse(login.context)
         self.client.get(reverse('admin:logout'))
 
+        # View User
```

Comment:
I'd prefer if you created separate test methods(e.g. `test_login_view_user`) for these additions as these test methods are already longer than normal.